rules/library.bzl

```diff
@@ -450,10 +524,58 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
             tags = _MANUAL,
         )
         vendored_deps.append(import_name)
+        import_headers = native.glob(
+            ["**/*.h"],
+            exclude = ["**/.DS_Store"],
+        )
+        import_module_maps = native.glob(
```

Comment:
nit: There should be no modulemap for a vendored_static_library, right?

Reply:
Yep it's quite common to include a module map with a vendored static library